charts/hivemq-platform/tests/configuration/configmap/hivemq_configuration_test.yaml

```
@@ -110,6 +109,20 @@ tests:
     asserts:
       - failedTemplate: {}
 
+  - it: with default values, only default labels created
+    asserts:
+      - isSubset:
+          path: metadata.labels
+          content:
+            helm.sh/chart: hivemq-platform-0.0.1
+            app.kubernetes.io/name: hivemq-platform
+            app.kubernetes.io/instance: test-hivemq-platform
+            app.kubernetes.io/version: 1.0.0
+            app.kubernetes.io/managed-by: Helm
+        template: hivemq-configuration.yml
+      - notExists:
+          path: metadata.annotations
```

`template` should go into the test level instead


```suggestion
 - it: with default values, only default labels created
    template: hivemq-configuration.yml
    asserts:
      - isSubset:
          path: metadata.labels
          content:
            helm.sh/chart: hivemq-platform-0.0.1
            app.kubernetes.io/name: hivemq-platform
            app.kubernetes.io/instance: test-hivemq-platform
            app.kubernetes.io/version: 1.0.0
            app.kubernetes.io/managed-by: Helm
      - notExists:
          path: metadata.annotations
```